$OpenBSD: patch-src_support_c,v 1.10 2012/12/05 14:31:43 dcoppa Exp $

Check for 'iTunes_Control' as well as 'iPod_Control', otherwise
gnome-mplayer won't detect some older iPod models
(upstream svn revision r2371)

--- src/support.c.orig	Wed Dec  5 15:58:22 2012
+++ src/support.c	Wed Dec  5 16:00:03 2012
@@ -2286,6 +2286,10 @@ gchar *find_gpod_mount_point()
             if (g_file_test(pathname, G_FILE_TEST_IS_DIR))
                 ret = g_strdup(mnt->mnt_dir);
             g_free(pathname);
+            pathname = g_strdup_printf("%s/iTunes_Control", mnt->mnt_dir);
+            if (g_file_test(pathname, G_FILE_TEST_IS_DIR))
+                ret = g_strdup(mnt->mnt_dir);
+            g_free(pathname);
         }
     }
     while (mnt);
